src/renderer/components/modals/create-note-modal/CreateNoteModal.html

```diff
@@ -7,27 +7,17 @@ <h2 class="text-red" v-if="displayDupError">Oh sorry, you can't have duplicated
 
     <section class="modal-card-body">
       <b-field v-if="!gistsSelected" horizontal label="Name">
-        <b-input
-          type="text"
-          ref="noteName"
-          v-model="note.name"
-          placeholder="Your note name">
+        <b-input type="text" ref="noteName" v-model="note.name" placeholder="Your note name">
```

Comment:
I think there's a formatting problem there

Reply:
Do you follow eslint config file in your editor configuration?

Reply:
Hi again, @lauthieb. I've spent some time trying stuff and I couldn't reproduce the bug.

Also, I ran `yarn eslint` & `yarn eslint:fix` and nothing got fixed. What should I do now?